### ISSUE TITLE
Another issue regarding lists and user data being used without permissions

### ISSUE THREAD

**Issue by stark1tty** [nathanlesage/academics-on-mastodon#106]
Hi,

I got a message from someone who went ahead and built an app using the data from this list, and then asked for permission afterwards. I did not give permission as I fear that this would violate GDPR. I'm raising the issue here to keep an eye on it. We have not been removed yet, however, it has only been a short time since I replied.

https://find-masto-folks.vercel.app/

I am thinking we should put a note in the readme explicitly stating to not remix these lists or use them in any way before gaining permission from list owners. It shouldn't need to be said, but it is getting rather frustrating... Thoughts?

Thanks.

**Comment by stark1tty** [comment]
I should note that this person seems honest and with best intent. We're chatting about GDPR, so we shall see. :)

**Comment by nathanlesage** [comment]
Heya, yeah, it's the main admin of sciences.social, and he has indeed good intents. I understand the issues one has with the GDPR, it's definitely a thick piece of legislation and takes some time getting used to.

However, he also contacted @trutzig89182 and me, and asked us for our opinions as well. Here, let me quickly reiterate my thoughts I sent him on his app:

* I think that offering people a more convenient way to utilize these lists rather than with CSV files is a great initiative, especially for the less technically versed people
* I mentioned that I don't think he needs to maintain his own database (because then you get into the swamp that is GDPR), and instead just download the current state of the lists inside the application (without too much technical yadda-yadda, downloading those lists within the app equates a single user downloading the lists onto their computer, which is fine). This would also have the added benefit that we have a single source of truth (the lists themselves) without any possibility of inconsistencies between his database and ours
* I also offered him that we could add a YAML file here in this repository where list maintainers can transparently opt in to this new method, and also have a machine-readable file for him to work with

In general, I think the balance between the opt-in way of handling such things and comfort is difficult to establish, but I think other than the other guy we had a few months ago, he has good intentions. I would be wary of adding yet another note to the README on what is not allowed, because I don't want to scare people away. By explicating more and more restrictions, we may make less-technical people afraid of doing something wrong, and I feel that the community should handle many of those issues itself, and I feel there's a good fostering of awareness towards these issues, where people are now beginning to pro-actively ask before actually implementing something like this.

What do you think?

**Comment by trutzig89182** [comment]
Hey @stark1tty, I understand your annoyance, I think it is a bit of a different situation with this. Mark did build this based on the interest to have something more practical for his own list and thought it might be interesting for other list maintainers. He did not make this link public (so it might make sense to remove it from your post above?) before first contacting us as the initial creators of the list-tool and then all other maintainers. It would still have been better to just demo what he suggests with his own list, but I trust that he will only put it out publicly once he has consent from the maintainers.

My positions was this: A better interface would be great, but I would oppose to

a) storing the information independently from the initial csv. My point would be less formally GDPR related, but rather: We put great effort to make sure we have consent for all users in the list and for me this means, that they won’t need to do something other than contacting us in order to be deleted from the app (although storing the csv in github is not a great solution independently from this specific issue).

b) somehow combining the information from the csv with data that is scraped and stored in a database.

I think the tool could be ok for me, if it is just another way of representing the data and offering another methods for following accounts on it (which might make it easier to follow people via smartphone). This would be the case if a) the user list is just mirroring the csv of the original list and b) account information is not stored permanently server side. @nathanlesage offered to offer a way of making the first part with the csv file possible. (Of course, only for those list maintainers who explicitly agree.)

For me the remaining question would then be, if the different way of representing the list would be sufficiently covered by the consent given by the list’s users. I would tend to say yes to this part, but am open for other opinions.

(P.S.: In my repository I have added a sentence that the csv is not covered by the foss lisence, when I added it –  for what it’s worth.)
